PopPUNK/network.py

```diff
@@ -279,7 +317,8 @@ def extractReferences(G, dbOrder, outPrefix, outSuffix = '', type_isolate = None
             sys.exit(1)
 
     if use_gpu:
-
+        if fast_mode:
+            raise RuntimeError("GPU graphs not yet supported with --update-db fast")
```

Comment:
This shouldn't be hard to implement in principle, but maybe we can create a general issue for a `network.py` update (including the function structures identified in #328) and work out the most efficient way to get this working across all modes clearly?